addons/web_dashboard/static/src/js/dashboard.js

```diff
@@ -0,0 +1,256 @@
+        this.load();
+        $('.loader').addClass("hidden");
+    },
+    show_loading:function(message){
+        $('.loader .text').text(_t(message));
+        $('.loader').removeClass('hidden');
+    }
+});
+
+var DashboardInvitations = Widget.extend({
+    template: 'DashboardInvitations',
+    events: {
+        'click .send_invitations': 'send_invitations',
+        'click .optional_message_toggler': 'optional_message_toggler',
+        'click .user': 'on_user_clicked',
+    },
+    init: function(parent, data){
+        this.data = data;
+        this.parent = parent;
+        this.user_form_id = this.parent.user_form_id;
+        return this._super.apply(this, arguments);
+    },
+    send_invitations:function(e){
+        var self = this;
+        var $target = $(e.currentTarget);
+        var user_emails = this.$el.find('#user_emails').val();
+        var optional_message = this.$el.find('#optional_message').val();
+        if(user_emails){
+            self.parent.show_loading("Sending Invitations");
+            $target.prop('disabled',true);
+            $target.find('i.fa-cog').removeClass('hidden');
+            ajax.jsonRpc("/dashboard/create_users", 'call', {
+                'users': user_emails,
+                'optional_message': optional_message
+            })
+            .then(function (data) {
+                $('.loader').addClass("hidden");
+                if(data.has_error){
+                    self.do_warn(data.message,"");
+                    $target.find('i.fa-cog').addClass('hidden');
+                    $target.attr('disabled',false);
+                }else{
+                    self.parent.do_reload();
+                }
+            });
+        }else{
+            this.do_warn(_t("Please enter email addresses"),"");
+        }
+
+    },
+    on_user_clicked: function (event) {
+        // event.preventDefault();
+        var user_id = $(event.target).data('user-id');
+
+        var action = {
+            type:'ir.actions.act_window',
+            view_type: 'form',
+            view_mode: 'form',
+            res_model: 'res.users',
+            views: [[this.user_form_id, 'form']],
+            res_id: user_id,
+        }
+        this.do_action(action);
+    },
+    optional_message_toggler:function(){
+        this.$el.find('.optional_message_toggler').remove();
+        this.$el.find('textarea.optional_message').slideToggle("fast");
+    }
+
+});
+
+var DashboardPlanner = Planner.PlannerLauncher.extend({
+    template: 'DashboardPlanner',
+     events: {
+        'click .proress_title,.progress': 'setup',
+    },
+    init: function(parent, data){
+        this.data = data;
+        this.parent = parent;
+        return this._super.apply(this, arguments);
+    },
+    start: function() {
+        var self = this;
+        return self.fetch_application_planner().done(function(apps) {
+            self.planner_apps = apps;
+            return apps;
+        });
+    },
+    setup: function(e){
```

Comment:
This is not a setup, this function aims to display the planner, right ? So rename it into `display_planner`.